src/libraries/System.Text.Json/tests/Common/JsonSchemaExporterTests.cs

```diff
@@ -90,6 +91,13 @@ public void UnsupportedType_ReturnsExpectedSchema(Type type)
             Assert.Equal(""""{"$comment":"Unsupported .NET type","not":true}"""", schema.ToJsonString());
         }
 
+        [Fact]
+        public void CanGenerateXElementSchema()
+        {
+            JsonNode schema = Serializer.DefaultOptions.GetJsonSchemaAsNode(typeof(XElement));
+            Assert.True(schema.ToJsonString().Length < 100_000);
```

Comment:
For context, the newly introduced schema reuse creates this monstrosity:

https://gist.github.com/eiriktsarpalis/fd53bb41b1f600a9b2e9c7985b9da64e

Reply:
That's before or after this PR?

Reply:
After the PR. Before the PR it just blows up.